### ISSUE TITLE
Compilation error after first installation

### ISSUE THREAD

**Issue by taniotanio7** [prescottprue/generator-react-firebase#172]
**Describe the bug**
After creating first project I get compilation errors.

```
Failed to compile.

./src/routes/Projects/components/ProjectsList/ProjectsList.js
  Line 36:20:  'firestore' is not defined  no-undef
  Line 68:11:  'isEmpty' is not defined    no-undef
```

**To Reproduce**
1. Installed generator
2. Switched to node version 10
3. Generate project in a directory (`yo react-firebase --skip-install`)
4. Install dependencies using yarn (`yarn install && yarn install --cwd functions`)
5. Start developent server (`yarn start`)

**Expected behavior/code**
Working project after installation.

**Possible Solution**
I guess that's a problem in the `ProjectsList.js` component... But why those methods (`firestore` and `isEmpty`) are missing... I have no clue.

**Additional Context/Screenshots**
.yo-rc.json file:
```json
{
  "generator-react-firebase": {
    "promptValues": {
      "githubUser": "taniotanio7",
      "firebaseName": "test-project-52a5d",
      "firebaseKey": "[REDACTED]",
      "includeRedux": false,
      "includeFirestore": true,
      "otherFeatures": [
        "Continuous Integration config",
        "Firebase Cloud Messaging",
        "Firebase Functions (with ESNext support)",
        "Tests for React Components (Jest)",
        "Tests for UI (Cypress)",
        "Firebase Analytics",
        "Client Side Error Reporting (Stackdriver)"
      ],
      "messagingSenderId": "[REDACTED]",
      "measurementId": "[REDACTED]",
      "appId": "[REDACTED]",
      "firebasePublicVapidKey": "[REDACTED]",
      "ciProvider": "travis",
      "deployTo": "firebase"
    }
  }
}
```

**Comment by prescottprue** [comment]
Thanks for reporting, I'll look into replicating soon

**Comment by prescottprue** [comment]
This should be fixed in v5.2.0
